themes/jquery/footer.php

```diff
@@ -36,7 +36,12 @@
 	</div>
 </footer>
 
-<?php wp_footer(); ?>
+<?php
+	wp_footer();
+
+	if ( function_exists( 'footer_append' ) )
+		footer_append();
```

Comment:
Okay, yes that works. Apologies, I must have had a type or something in my tests.